Gems/EMotionFX/Code/EMotionFX/Source/BlendTreeTransformNode.cpp

```diff
@@ -261,23 +261,23 @@ namespace EMotionFX
             ->Attribute(AZ::Edit::Attributes::ChangeNotify, &BlendTreeTransformNode::Reinit)
             ->Attribute(AZ::Edit::Attributes::ChangeNotify, AZ::Edit::PropertyRefreshLevels::EntireTree)
             ->DataElement(AZ::Edit::UIHandlers::Default, &BlendTreeTransformNode::m_minTranslation, "Min Translation", "The minimum translation value, used when the input translation amount equals zero.")
-            ->Attribute(AZ::Edit::Attributes::Min, AZ::Vector3(-std::numeric_limits<float>::max()))
-            ->Attribute(AZ::Edit::Attributes::Max, AZ::Vector3(std::numeric_limits<float>::max()))
+            ->Attribute(AZ::Edit::Attributes::Min, -std::numeric_limits<float>::max())
+            ->Attribute(AZ::Edit::Attributes::Max, std::numeric_limits<float>::max())
```

Comment:
Just curious, why put min/max at all in this case, where it covers the entire value range?  Does it do anything different than *not* having them?

Reply:
I'm also curious about that one :). If not, we can just remove them.

Reply:
mm good point I didn't want to change the original intent like maybe if Qt had different limits by default, but yeah let me test it out.

Reply:
OK so learned something I didn't know/wasn't expecting, which is the default minimum is 0.0 and the default maximum is 99.99:    https://doc.qt.io/qt-5/qdoublespinbox.html#minimum-prop

So turns out this is needed :)  Although this raises another question which is maybe we should have our spinbox have a default min/max that would allow all values?

Reply:
Ignore me on that last comment, I found the place where it does set the min/max to wide range:

```
VectorPropertyHandlerCommon::ConstructGUI

newCtrl->setMinimum(-std::numeric_limits<float>::max());
newCtrl->setMaximum(std::numeric_limits<float>::max());
```

So looks like yeah I can remove those min/max that set the same limit.